Rocket.Chat.iOSUITests/Rocket_Chat_iOSUITests.swift

```diff
@@ -8,7 +8,6 @@
 
 import XCTest
 
-@available(iOS 9.0, *)
```

Comment:
I will add this, this probably happened when I merged accidentally and then reverted